roadmap/implementors-guide/guide.md

```diff
@@ -1001,9 +1019,27 @@ Dispatch a `PovFetchSubsystemMessage(relay_parent, candidate_hash, sender)` and
 
 (TODO: send statements to Statement Distribution subsystem, handle shutdown signal from candidate backing subsystem)
 
+### Statement Distribution Subsystem
+
+The statement distribution subsystem sends statements to peer nodes, detects double-voting, and tabulates when a sufficient portion of the validator set has unanimously judged a candidate. When judgment is not unanimous, it escalates the issue to misbehavior arbitration.
+
+Statement Distribution is the only subsystem which has any direction notion of peer nodes or of our own cryptographic keys. It is responsible for deciding when a quorum exists, and for detecting a variety of peer node misbehaviors for reporting to Misbehavior Arbitration. It's the main point of contact (via the Overseer) with peer nodes. It also responds to secondment by peer nodes by forwarding those votes to validation to double-check the peers.
```

Comment:
> > It is responsible for deciding when a quorum exists
> I would expect this to be the responsibility of Candidate Backing

Candidate backing, as things stand right now, needs only to consider the set of candidate blocks for a given parachain and relay head, in order to choose one of them. It doesn't need to concern itself with any other peer node's view of the situation.

Statement Distribution, on the other hand, naturally needs to know who the peer nodes are, and `SignedStatement`s from each of them will pass through this subsystem. It would be straightforward to tally the votes within that subsystem.

> It seems likely that we'll have many other subsystems that perform networking actions. 

Yes, fair enough. What that should say is something like "Within the Validity module, comprising the subsystems defined in https://github.com/paritytech/polkadot/pull/1161, it's the main point of contact etc." The idea is that within this module, we can isolate that responsibility; of course I don't mean to forbid networking actions from other modules.

Reply:
> What does forwarding to validation mean in this context?

Incoming `SignedStatement`s come in from peer validators to Statement Distribution. Statement Distribution can handle the basic work of checking those statements' signatures, but that's not sufficient. 

The basic cases for validating a block are when Candidate Backing thinks we should second a block, and double-checking when a peer validator has seconded one. All that sentence means is that when Statement Distribution receives an appropriately signed `Statement::Second` from a peer, it then sends the block to Candidate Validation to double-check the peer's judgment.

Reply:
I've moved the responsibility for quorum decisions to Candidate Backing.